torchprime/launcher/cli.py

```diff
@@ -77,6 +79,54 @@ def cli(ctx, interactive):
   ctx.obj["interactive"] = interactive
 
 
+@cli.command("save-hf-model-files-to-gcs")
```

Comment:
probably you should submit your PR for removing click dependency first and then update this one